src/js/components/SkipLinks/SkipLinks.js

```diff
@@ -35,14 +34,21 @@ const SkipLinks = ({ children, id, messages }) => {
   return (
     <Layer
       id={id}
-      position={showLayer ? 'top' : 'hidden'}
+      position={showLayer ? theme.skipLinks.position : 'hidden'}
       ref={layerRef}
       onFocus={onFocus}
       onBlur={onBlur}
+      modal={false}
+      targetChildPosition="first" // prepend the Layer on the body container
```

Comment:
Can we add just a bit more to this comment to indicate why we are doing this? For example:

"Prepend the Layer so any SkipLink will be the first element that pressing the Tab key reaches"

Reply:
Absolutely. good comment.